### DISCUSSION TITLE
Quasar Build - How to configure shell command in afterBuild?

### DISCUSSION THREAD

**Discussion by rodrigoslayertech** [quasarframework/quasar#14212] [General - Components / Directives / etc]
Hello,
I would like to configure a shell command every time the `quasar build` ends.

Currently, I can do this in the terminal with this command:
`quasar build; rm -rf dist/public/; cp -r dist/spa/ dist/public/`

I have a custom web server that checks if the index.html exists in the `dist/spa` folder, if it does not exist, it uses the `dist/public` as backup folder. Because during the build the `dist/spa` folder is temporarily deleted and the website is temporarily offline too.

How to configure this in `afterBuild` hook? What's the best practices to keep security?

```
# quasar.config.js

build: {
   ...
   afterBuild: ??
   ...
}
```

**Comment by MilosPaunovic** [comment]
```js
afterBuild({ quasarConf }) => {
  // run Node code for what you need
}
```
